.github/workflows/bow-php-action.yml

```diff
@@ -47,9 +44,6 @@ jobs:
 
       - run: docker run -p 1025:25 -p 1080:80 -d --name maildev maildev/maildev
 
-      - name: Validate composer.json and composer.lock
-        run: composer validate --strict
```

Comment:
This `composer validate` command will be always failed because the `composer.lock` is outdated.

Remove this step and using the `composer update` command to update and install the required dependencies.